.circleci/config.yml.j2

```diff
@@ -131,13 +131,25 @@ jobs:
           when: always
 {%- endfor %}
 
+  check-commander-airflow-version:
+    docker:
+      # https://hub.docker.com/_/docker?tab=tags
+      - image: cimg/base:2021.04
```

Comment:
Any reason why we're using a year old tag? Could we use `2022.03` ?

https://circleci.com/developer/images/image/cimg/base#image-tags

Reply:
Took it from CircleCI docs, I'll update to latest one, thanks for that catch